config/core/resources/configuration.yaml

```diff
@@ -22,16 +22,25 @@ metadata:
     duck.knative.dev/podspecable: "true"
 spec:
   group: serving.knative.dev
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      # this is a work around so we don't need to flush out the
+      # schema for each version at this time
+      #
+      # see issue: https://github.com/knative/serving/issues/912
+      x-kubernetes-preserve-unknown-fields: true
   versions:
   - name: v1alpha1
     served: true
-    storage: true
+    storage: false
```

Comment:
I think we want to leave this as the storage version for a release.

Reply:
reverted the storage version back to `v1alpha1`

Reply:
Issue is here: https://github.com/knative/serving/issues/6726